samples/notifications/ToastNotificationSample/ToastNotificationSample/Activation/DefaultLaunchActivationHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using ToastNotificationSample.Helpers;
+using ToastNotificationSample.Services;
+
+using Windows.ApplicationModel.Activation;
+
+namespace ToastNotificationSample.Activation
+{
+    internal class DefaultLaunchActivationHandler : ActivationHandler<LaunchActivatedEventArgs>
+    {
+        private readonly Type _navElement;
+
+        public DefaultLaunchActivationHandler(Type navElement)
+        {
+            _navElement = navElement;
+        }
+
+        protected override async Task HandleInternalAsync(LaunchActivatedEventArgs args)
+        {
+            // When the navigation stack isn't restored, navigate to the first page and configure
+            // the new page by passing required information in the navigation parameter
+            NavigationService.Navigate(_navElement, args.Arguments);
+
+            // TODO WTS: This is a sample on how to show a toast notification.
```

Comment:
Should differentiate the comments added to the sample from the "TODO WTS" comments added as part of generation. The sample doesn't show things that need to be done but rather things that have been done.